### PR TITLE
Added ANI-1ccx model with a similar loader to Builtins.

### PR DESCRIPTION
**Pull request by LeanAndMean**

### CODE REVIEW

File: torchani/neurochem/__init__.py [hunk 1]
```diff
@@ -275,27 +282,91 @@ class Builtins:
...
         self.ensemble_prefix = pkg_resources.resource_filename(
-            parent_name, 'resources/ani-1x_dft_x8ens/train')
+            parent_name,
+            ensemble_prefix_path)
         self.models = load_model_ensemble(self.consts.species,
                                           self.ensemble_prefix,
                                           self.ensemble_size)
 
+class Builtins(BuiltinsAbstract):
+    """Container for all builtin stuffs.
```
Comment (zasdfgbnm) [inline torchani/neurochem/__init__.py]:
```suggestion
    """Container for the builtin ANI-1x model.
```

File: torchani/neurochem/__init__.py [hunk 2]
```diff
@@ -275,27 +282,91 @@ class Builtins:
...
+            parent_name,
+            const_file_path,
+            sae_file_path,
+            ensemble_size,
+            ensemble_prefix_path
+        )
+
+
+class BuiltinsANI1CCX(BuiltinsAbstract):
+    """Container for all builtin stuffs.
```
Comment (zasdfgbnm) [inline torchani/neurochem/__init__.py]:
```suggestion
    """Container for the builtin ANI-1ccx model.
```

**Comment by isayev** [comment]
Fellas, great feature! However, building a separate class for each new type of net is not very handy. How about defining a dictionary of available sets and then pass its high-level name (e.g. ani1x, ani2x, ani1ccx etc) to a class?

**Comment by zasdfgbnm** [comment]
@isayev I am personally OK with creating a class for each model, but open for discussion. torchvision is doing the same way:
https://github.com/pytorch/vision/tree/master/torchvision/models

**Comment by zasdfgbnm** [comment]
@isayev Maybe in the future, when we get more models, we could have a `torchani.models` subpackages that host all builtin models, used as (`torchani.models.ANI1x`)

**Comment by isayev** [comment]
Sure, I am personally happy to write either. However I am thinking about
broad base if users. Will you guybe able to make a quick example/notebook
showcasing how to swipe different nets.

**Comment by zasdfgbnm** [comment]
@isayev Maybe put it here?
https://aiqm.github.io/torchani/examples/

**Comment by zasdfgbnm** [comment]
The builds are successful, but I don't know why Azure didn't report the status here. I will reopen this PR and let it run agian.

**Comment by zasdfgbnm** [comment]
@LeanAndMean Could you fix the linter error?

**Pull request merged**